jsapp/js/components/formGallery/formGallery.component.scss

```diff
@@ -0,0 +1,66 @@
+@use 'scss/_colors';
+
+.gallery {
+  background-color: colors.$kobo-white;
+}
+
+.gallery__wrapper {
+  margin-top: 20px;
+  margin-left: 54px;
+  margin-right: 54px;
+
+  @media screen and (max-width: 768px) {
+    margin-top: 10px;
+    margin-left: 15px;
+    margin-right: 15px;
+  }
+
+  .gallery-bottom {
+    width: 100%;
+    display: flex;
+    justify-content: center;
+    justify-content: center;
+    display: flex;
+  }
+}
+
+.gallery__header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  border-bottom: 1px solid colors.$kobo-gray-92;
+}
+
+.gallery__icons {
+  font-size: 24px;
+}
+
+.gallery__icons i {
```

Comment:
I tested this out with multiple <button>'s. Works with one additional display: flex to prevent stacking vertically. `font-size: 24px;` no longer seems to do anything either.